include/log.h

```diff
@@ -83,6 +83,10 @@
 # include <login.h>
 #endif
 
+#ifdef HAVE_EXECINFO_H
+# include <execinfo.h>
+#endif
```

Comment:
You probably don't need this `#include`, since the function signatures don't use any of the data types from this header.